tests/test_graphql_farmers.py

```
@@ -0,0 +1,136 @@
+import pytest
+from httpx import AsyncClient
+
+from app.main import app
```

Ruff (0.12.2)

`app.main.app` imported but unused

Remove unused import: `app.main.app`

(F401)